package.json

```diff
@@ -211,7 +212,8 @@
     "**/@types/react": "^18.0.9",
     "**/@types/react-is": "^17.0.3",
     "**/cross-fetch": "^3.1.5",
-    "**/react-is": "^18.2.0"
+    "**/react-is": "^18.2.0",
+    "**/thenify": "^3.3.1"
```

Comment:
```suggestion
    // thenify@3.3.1 has a fix for security vulnerability, see https://github.com/mui/material-ui/security/dependabot/78
    "**/thenify": "^3.3.1"
```

Reply:
cc @siriwatknp let's add a comment as to why this resolution is made.

Reply:
🥲 I don't think we can add a comment in json.